ui.frontend/src/components/CodeArgumentInput.tsx

```diff
@@ -90,9 +90,11 @@ const CodeArgumentInput: React.FC<CodeArgumentInputProps> = ({ arg }) => {
                   maxValue={arg.max !== null ? Dates.toCalendarDate(arg.max) : undefined}
                   onChange={(dateValue) => field.onChange(dateValue?.toString())}
                   label={label}
+                  {...(fieldState.error && {
```

Comment:
for me previous syntax was more readable

Reply:
it's commong patetrn among react developers, and secondnly we are getting rid of 'valid' status before validation